### DISCUSSION TITLE
Uploading iPad pro screenshots

### DISCUSSION THREAD

**Discussion by chrisoshea** [fastlane/fastlane#17267] [Q&A] [answered]
On iOS, the required images for iPad are iPad Pro (3rd gen) and iPad Pro 9 (2nd gen). Both uploads are required, you can't tick gen 2 to use gen 3 screenshots in App Store Connect. Both are the same size 2732x2048

Deliver has trouble uploading to both to the right place. When you download the existing screenshots using Deliver, you get files like this:

0_APP_IPAD_PRO_3GEN_129_0.png
0_APP_IPAD_PRO_129_0.png

However when you reupload using Deliver and keep the same file names, it puts all of the iPad screenshots under 2nd gen and none in 3rd gen (so you get duplicates).

Any thoughts about this? Tagging @joshdholtz as per twitter request.

Thanks

**Comment by ainame** [comment, marked as answer]
I think this is going to resolve your issue but it hasn't got merged yet.

https://github.com/fastlane/fastlane/pull/17364
